app/domain/authentication/authn_k8s/copy_text_to_file_in_container.rb

```diff
@@ -42,29 +42,28 @@ def copy_text_to_file_in_container
       #
       # We redirect the output to a log file on the authn-client container
       # that will be written in its logs for supportability.
-      def set_file_content_script(path, content, mode)
+      def bash_script(path, content, mode)
```

Comment:
I agree with @Tovli that the script should be in external file.

Reply:
i disagree on this. reading a script from an external file adds redundant complexity. Having it in the same file has better readability

Reply:
A key question: Is the script intended to be used any place _other_ than in the ruby file?  If the answer is no, keeping it here seems convenient, since it avoids some boilerplate around reading the file from disk and then doing the cert interpolation on that read-in string.

You lose syntax highlighting when reading it, but since the script is so short and simple this seems like a reasonable price to pay.

Reply:
it is only used as part of this ruby file. it is a key part of this class and this class only.